logger.go

```diff
@@ -138,39 +137,21 @@ type zapAdapter struct {
 
 var _ Logger = &zapAdapter{}
 
-// createdScopes tracks the scopes that have been created so far.
-var createdScopes sync.Map
```

Comment:
I think once upon a time there was an idea to have a debug endpoint where you can take a scope and look up "what's this scope about"